src/command_modules/azure-cli-dls/HISTORY.rst

```diff
@@ -2,9 +2,14 @@
 
 Release History
 ===============
+0.0.12
```

Comment:
This should be 'unreleased'
https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log